vibe_code_apps_22/notebooks/lakeside_desktop_sim_playground.ipynb

```
@@ -83,69 +86,115 @@
+    "def _load_peak_day(path: Path, prefer: str | None = None) -> tuple[pd.DataFrame, object]:\n",
+    "    df = pd.read_parquet(path)\n",
+    "    df = df.copy()\n",
+    "    df[\"timestamp_local\"] = pd.to_datetime(df[\"timestamp_local\"])\n",
+    "    daily = (\n",
+    "        df.groupby(df[\"timestamp_local\"].dt.date)[\"facility_kw\"]\n",
+    "        .agg(peak=\"max\", n=\"count\")\n",
+    "        .query(\"n >= 90\")\n",
+    "        .sort_values(\"peak\", ascending=False)\n",
+    "    )\n",
+    "    if daily.empty:\n",
+    "        raise RuntimeError(f\"no complete days in {path}\")\n",
```

Ruff (0.16.1)

[error] Multiple statements on one line (semicolon)

(E702)